modules/odr_monitoring/odr_monitoring/handlers/console_handler.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import sys
+from ..config import logging_config
```

Comment:
can we shift this to absolute module paths? i.e odr_monitoring.config , odr_monitoring.formatters.standard_formatter, etc

Reply:
Sure thing, I'll make the change